packages/dnb-eufemia/src/extensions/forms/Iterate/ArrayRemoveElementButton.tsx

```diff
@@ -0,0 +1,75 @@
+import React, { useCallback, useContext } from 'react'
+import classnames from 'classnames'
+import { Button } from '../../../components'
+import { ButtonProps } from '../../../components/Button'
+import { forwardSpaceProps } from '../utils'
+import IterateElementContext from './IterateElementContext'
+import { useDataValue } from '../hooks'
+import {
+  ComponentProps,
+  DataValueReadProps,
+  DataValueWriteProps,
+} from '../types'
+
+export type Props = ComponentProps &
+  DataValueReadProps<unknown[]> &
+  DataValueWriteProps<unknown[]> & {
+    // Button props
+    variant?: ButtonProps['variant']
+    size?: ButtonProps['size']
+    text?: ButtonProps['text']
+    icon?: ButtonProps['icon']
+    iconPosition?: ButtonProps['icon_position']
+    iconSize?: ButtonProps['icon_size']
+    title?: ButtonProps['title']
+    children?: ButtonProps['children']
+  }
+
+function ArrayRemoveElementButton(props: Props) {
+  const iterateElementContext = useContext(IterateElementContext)
+  const { handleRemove } = iterateElementContext ?? {}
+
+  if (!iterateElementContext) {
+    throw new Error(
+      'ArrayRemoveElementButton must be inside an Iterate.Array component.',
+    )
+  }
+
+  const {
+    variant,
+    size,
+    text,
+    icon,
+    iconPosition,
+    iconSize,
+    title,
+    children,
```

Comment:
```suggestion
    children,
    ...rest
```

So we can send along HTML attributes like `aria-*`.

Reply:
Fixed in last commit: https://github.com/dnbexperience/eufemia/pull/2635/commits/ad80ce33d1015241763351781b323d7b0eb725c6